### ISSUE TITLE
Cobertura line-rate is always 1

### ISSUE THREAD

**Issue by praxamarnix** [GioCirque/dart-pub#5]
I noticed that using the tool, I always get a line-rate of 1 for my current classes. When I look through the code, I see that the line-rate is always rounded and I don't really understand why that is, since the line-rate should be a ratio between 0 and 1.

For example, I have:

```
SF:lib\dependency_injection.dart
DA:22,1
DA:23,1
DA:24,1
DA:25,1
DA:26,3
DA:31,1
DA:32,3
DA:34,3
DA:35,2
DA:36,2
DA:39,3
DA:40,2
DA:41,2
DA:42,2
DA:45,2
DA:46,3
DA:48,2
DA:49,2
DA:51,1
DA:52,1
DA:53,1
DA:54,1
DA:56,2
DA:57,2
DA:62,0
DA:63,0
DA:64,0
DA:68,0
DA:69,0
LF:29
LH:24
end_of_record
```

https://github.com/GioCirque/pub-dev/blob/main/cobertura/bin/src/cobertura.dart#L138 shows us that it will become (24/29).round()=1. Which I don't think would be correct.

I'm not sure if this tool is still in development, but I was hoping to use it.

**Comment by VannaDii** [comment]
Wow, sorry, I haven't seen this issue until now. I'm doing a round of fixes on several libs and will look into this. Yeah, that shouldn't be rounding, tho.

**Comment by VannaDii** [comment]
This is resolved in v1.0.4
